hcx-pipeline-jobs/core/src/main/scala/org/swasth/dp/core/util/DispatcherUtil.scala

```diff
@@ -75,9 +75,9 @@ object DispatcherUtil {
       } else  //As url is null, no need to retry
         DispatcherResult(false, 0, None, false)
     } catch {
-      case ex: Exception => {
-        DispatcherResult(false, 0, None, true)
-      }
+      case ex: Exception =>
+        val errorResponse: ErrorResponse = ErrorResponse(Option(Constants.RECIPIENT_ERROR_CODE), Option(Constants.RECIPIENT_ERROR_MESSAGE), Option(""))
```

Comment:
No need of sending errorResponse as we will be retrying

Reply:
when the max retry limit is reached, errorResponse is required in dispatching the error to sender.